### PR TITLE
(tlg0008).tlg0008.tlg001-Updating _cts_.xml file, headers, and bibliographic data

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Added missing perseus-eng3 into the _cts_.xml file
Updated headers and bibliographic descriptions
Added in reference targets.

### CODE REVIEW

File: data/tlg0008/tlg001/tlg0008.tlg001.perseus-eng2.xml [hunk 1]
```diff
@@ -31,11 +30,16 @@
...
                   <title>The Deipnosophists</title>
-                  <title>Or Banquet Of The Learned Of Athenaeus.</title>
+                  <title type="sub">Or Banquet Of The Learned Of Athenaeus</title>
+               <editor role="translator">Charles Duke Yonge</editor>
                      <pubPlace>London</pubPlace>
-                     <publisher>Henry G. Bohn, York Street, Covent Garden.</publisher>
+                     <publisher>Henry G. Bohn</publisher>
                      <date>1854</date>
-            </bibl>
+               <biblScope type="volume">1-3</biblScope>
```
Comment (lcerrato) [inline data/tlg0008/tlg001/tlg0008.tlg001.perseus-eng2.xml:38]: <biblScope unit="volume">

File: data/tlg0008/tlg001/tlg0008.tlg001.perseus-eng3.xml [hunk 2]
```diff
@@ -35,16 +34,17 @@
...
+							</editor>
 							<imprint>
 								<pubPlace>Cambridge, MA</pubPlace>
 								<publisher>Harvard University Press</publisher>
 								<pubPlace>London</pubPlace>
 								<publisher>William Heinemann Ltd.</publisher>
-								<date>1927</date>
+								<date>1927-1941</date>
 							</imprint>
 							<biblScope type="volume">1-7</biblScope>
```
Comment (lcerrato) [inline data/tlg0008/tlg001/tlg0008.tlg001.perseus-eng3.xml:49]: <biblScope unit="volume">

**Review by lcerrato — CHANGES_REQUESTED** [review]
@AlisonBabeu 
One of these files no longer passes, and I think that the biblScope issue (below) is the sole cause.

Something to be aware of is that these files point to a specific version of the schema — this is n't standard practice now. We always point to the latest version rather than a specific version.

This should not be resolved in these files now — it's good to leave as is because it means these need another check. (Also, I think we want Monica's Greek for these files anyhow so I have not prioritized these.) But it is something to be aware of because files with the old schema may appear to validate offline but not validate with the tests (which point to the newer schema).

File: data/tlg0008/tlg001/tlg0008.tlg001.perseus-eng3.xml (hunk 2 above)
Comment (AlisonBabeu) [reply data/tlg0008/tlg001/tlg0008.tlg001.perseus-eng3.xml:49]: Hey @lcerrato so to resolve this do I need to change back the biblScope in these files to what it was previously? I'm afraid I'm not sure how exactly what I need to change  to make these files pass.  I did notice about the schema issue, but hoped that if I was only changing headers and other basic metadata that it wouldn't be an issue.   Sorry if I created an unnecessary issue.

Comment (AlisonBabeu) [reply data/tlg0008/tlg001/tlg0008.tlg001.perseus-eng3.xml:49]: How did I not see that? Thanks and I will change that!

**Review by lcerrato — APPROVED** [review]

**Pull request merged**